lib/src/server.dart

```diff
@@ -170,7 +173,8 @@ class Server {
         'result': result,
         'id': request['id']
       };
-    }).catchError((error, stackTrace) {
+    } catch (e, stackTrace) {
+      var error = e;
```

Comment:
Done. After pulling the `return null` branch out; it seemed simpler to use a ternary for the two cases and assign to a temporary variable.